app/src/main/java/module2/globant/calculator/mvp/model/CalculatorModel.java

```diff
@@ -0,0 +1,27 @@
+package module2.globant.calculator.mvp.model;
+
+import module2.globant.calculator.Constants.Constants;
+
+public class CalculatorModel {
+
+    private int operation = Constants.SUM;
+
+    public void setOperation(int operation) {
+        this.operation = operation;
+    }
+
+    public int getOperation() {
+        return operation;
+    }
+
+    public double doOperation(int operation, double a, double b) {
```

Comment:
why do you have operation as parameter if you already have it as attribute, remove operation as attribute or remove it as parameter.